src/chess.ts

```diff
@@ -101,6 +101,7 @@ const FLAGS: Record<string, string> = {
   PROMOTION: 'p',
   KSIDE_CASTLE: 'k',
   QSIDE_CASTLE: 'q',
+  NULL_MOVE: 'nm',
```

Comment:
The FLAGS.NULL_MOVE value should be a single character. As written, the `n` in `nm` collides with FLAGS.NORMAL. Maybe use '-'? 

The whole flag system is an artifact of some C code I wrote years ago, and is kinda kludgy.

Reply:
I fixed this with the suggested value '-'